common/djangoapps/student/helpers.py

```diff
@@ -230,3 +235,38 @@ def get_next_url_for_login_page(request):
         # be saved in the session as part of the pipeline state. That URL will take priority
         # over this one.
     return redirect_to
+
+
+def get_profile_visibility(user_profile, user, configuration=None):
```

Comment:
I moved it here because it is where the origin data for user profiles is stored. So it seemed like the semantically correct place to put it. I could move it there instead.